features/steps/proxy_component.go

```diff
@@ -42,9 +43,11 @@ func NewComponent() (*Component, error) {
 
 	c.babbageFeature = NewBabbageFeature()
 	c.legacyCacheAPIFeature = NewLegacyCacheAPIFeature()
+	c.releaseCalendarFeature = NewReleaseCalendarFeature()
 
 	c.Config.BabbageURL = c.babbageFeature.Server.URL
 	c.Config.LegacyCacheAPIURL = c.legacyCacheAPIFeature.Server.URL
+	c.Config.RelCalURL = c.releaseCalendarFeature.Server.URL
```

Comment:
You also need to close the server in the `Close` function further down in this file.